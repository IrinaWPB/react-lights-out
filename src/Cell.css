.Cell {
  height: 100px;
  width: 100px;
  background-color: rgb(94, 94, 94);
  cursor: pointer;
  border-radius: 50%;
}

.Cell-lit {
  background-color: rgb(255, 251, 131);
  box-shadow: 0 0 10px rgb(157, 157, 1);
}