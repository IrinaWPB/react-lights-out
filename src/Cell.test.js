import React from 'react'
import { render } from '@testing-library/react'
import Cell from './Cell'

it ('renders without crashing', () => {
    const { asFragment } = render(<table><tbody><tr><Cell /></tr></tbody></table>)
    expect(asFragment()).toMatchSnapshot()
})