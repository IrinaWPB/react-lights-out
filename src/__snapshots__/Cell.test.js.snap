// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <table>
    <tbody>
      <tr>
        <td
          class="Cell "
        />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
